source/code/plugins/filter_baseline.rb

```diff
@@ -0,0 +1,38 @@
+ 

```

Comment:
General: consider adding a comment that this filter creates both the baseline and the baseline_summary records